APMrover2/GCS_Mavlink.cpp

```diff
@@ -858,7 +858,11 @@ void GCS_MAVLINK_Rover::handleMessage(mavlink_message_t* msg)
             }
             result = MAV_RESULT_ACCEPTED;
             break;
-#endif  // CAMERA == ENABLED
+
+        case MAV_CMD_DO_SET_CAM_TRIGG_DIST:
+            copter.camera.set_trigger_distance(packet.param1);
```

Comment:
There isn't a copter in rover. Same issue exists in plane. Please try compiling all the build targets if you change code in any of them.